test/Application.UTest/Games/UpdateGameUsersCommandTest.cs

```diff
@@ -18,7 +18,8 @@ public void ShouldDoNothingForEmptyUpdates()
     {
         Mock<ICharacterService> characterServiceMock = new();
         Mock<IActivityLogService> activityLogServiceMock = new() { DefaultValue = DefaultValue.Mock };
-        UpdateGameUsersCommand.Handler handler = new(ActDb, Mapper, characterServiceMock.Object, activityLogServiceMock.Object);
+        Mock<CompetitiveRatingModel> competitiveRatingModelMock = new() { DefaultValue = DefaultValue.Mock };

```

Comment:
The `DefaultValue` property is not needed here, you could remove it.

Reply:
done